app/models/setting.rb

```diff
@@ -286,7 +286,7 @@ def self.column_check(opts)
   # End methods for loading default settings
 
   def full_name_with_default
-    "#{full_name} (Default: #{default})"
+    _("%{full_name} (Default: %{default})") % {full_name: _(full_name), default: default}
```

Comment:
do you need to have the translation for `_(full_name)` if the whole thing is translated?

Reply:
yes, the full_name is extracted separately ins the setting definition. the translation for the whole string is actually just to get the default translated in context correctly